src/swap/SwapScreen.test.tsx

```diff
@@ -91,7 +91,11 @@ const renderScreen = ({ celoBalance = '10', cUSDBalance = '20.456' }) => {
 
   const tree = render(
     <Provider store={store}>
-      <SwapScreen />
+      {showDrawerTopNav ? (
+        <SwapScreen />
+      ) : (
+        <SwapScreenSection showDrawerTopNav={showDrawerTopNav} />
+      )}
```

Comment:
not sure I understand this. Wouldn't this mean that for the else part, `showDrawerTopNav` is always false?

Reply:
Updated to remove the conditional and just render the `SwapScreenSection`